### DISCUSSION TITLE
How to reuse existing styles in extensions ?

### DISCUSSION THREAD

**Discussion by VaZark** [directus/directus#11651] [Archive]
I'm currently creating a module as prototype for a feature request I raised in the discussion on #11635  .

I've caught on to the basics but I need some input on how to reuse styling in the App for modules extensions. Shown below in the screenshot is the state of my module with bare basic styling. Docs do not seem to have info regarding it though. Do I have to dive into the code? If so, I can help write up documentation regarding the same.

![image](https://user-images.githubusercontent.com/11616688/154088622-d7b1157e-ae47-405b-8612-2595dfb2b456.png)

**Comment by rijkvanzanten** [comment]
Hey! You can use any of the globally registered base components in custom extensions as well. It's not the most in-depth reference, but there's [more information available on the wiki](https://github.com/directus/directus/wiki/app-components) 🙂
